components/ShoppingItem.js

```diff
@@ -61,7 +61,7 @@ const ItemContainer = styled.li`
     "itemquantity itemcategory mark delete";
   border: 1px solid #362f23;
   padding: 0;
-  margin: 6px;
+  margin: 0 6px 6px 6px;
```

Comment:
there is a shorthand for this: 
```suggestion
  margin: 0 6px 6px;
```
the second "6px" is applied to the left and right border of the element.

Reply:
Thanks! I wasn't aware of this shortcut.